tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor/Utils/RulesConfiguration.cs

```diff
@@ -47,9 +49,17 @@ public RulesConfiguration(string configurationFile = null)
             string configLoc = configurationFile;
             if (configLoc == null)
             {
-                // Load the config from the well known location, somewhere under the .github directory
-                // which is in the root of the repository
-                configLoc = DirectoryUtils.FindFileInRepository(RulesConfigFileName, RulesConfigSubDirectory);
+                if (null != rulesConfigFilePathOverride)
+                {
+                    // If the user overrode the location
+                    configLoc = rulesConfigFilePathOverride;
+                }
+                else
+                {
+                    // Load the config from the well known location, somewhere under the .github directory
+                    // which is in the root of the repository
+                    configLoc = DirectoryUtils.FindFileInRepository(RulesConfigFileName, RulesConfigSubDirectory);
```

Comment:
Is it expect that folks would still use local files if they want to test new configuration files before updating? I was trying to figure out how we would validate any changes to these files in a language repo. I don't think there is any great way to validate them in the CI but it would be nice to have a way to at least run them through the tool before blindly taking changes to them in the language repo.